main.nf

```diff
@@ -124,6 +124,12 @@ bam_bqsr_choice.map {
 // 	split_vep_choice;
 // }
 
+Channel
+	.fromPath(params.csv)
+	.splitCsv(header:true)
+	.map{ row-> tuple(row.group, row.assay) }
+        .set{ meta_loqusdb_no_sv_calling }
```

Comment:
`assay` here is not required by the process, other than to disable a nextflow warning about a channel emitting only one input to the new process below